images/kubernetesonarm/elasticsearch/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM larmog/armhf-alpine-java:jdk-8u73
+
+EXPOSE 9200 9300
```

Comment:
The 9200 port is the default es port and 9300 is for node discovery in a es cluster. You need them.